src/main/java/com/minecolonies/coremod/achievements/MineColoniesAchievementPage.java

```diff
@@ -10,17 +9,177 @@
  *
  * @since 0.2
  */
-public class MineColoniesAchievementPage extends AchievementPage
+public class MineColoniesAchievementPage
 {
+    //todo have to extend advancement at some point
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1135)